Game_Manual/Echo.md

```diff
@@ -18,4 +18,47 @@ For many commands, passing the `-h` or `--help` flag will show you a helpful mes
 
 For example, to get help with the `/review` command, type `/review --help`.
 
-[los-repo]: https://github.com/LearnersGuild/los
+It's also the primary means by which we communicate with each other - we don't send email at Learners Guild. Because we have so many people communicating on a public platform, it's important to observe some shared Guidelines:
```

Comment:
This feels like it's missing an explanatory sentence before?  `It's also` implies it's something else that was just described, and I don't see what that is?